src/current/v23.1/advanced-changefeed-configuration.md

```diff
@@ -70,30 +70,51 @@ Use the following workflow to enable `MuxRangefeed`:
+{{site.data.alerts.callout_info}}
+Adjusting `kv.closed_timestamp.target_duration` could have a detrimental impact on [follower reads]({% link {{ page.version.version }}/follower-reads.md %}). If you are using follower reads, refer to the [`kv.rangefeed.closed_timestamp_refresh_interval`](#kv-rangefeed-closed_timestamp_refresh_interval) cluster setting instead to ease changefeed impact on foreground SQL latency.
+{{site.data.alerts.end}}
 
-In clusters running large-scale workloads, increasing this setting can help to lower the potential impact of changefeeds on SQL latency. That is, an increase in the setting could lower the load on the cluster. This is important for workloads with tables in the TB range of data. However, for most workloads, we recommend leaving this setting at the default of `3s`.
+`kv.closed_timestamp.target_duration` controls the target [closed timestamp]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#closed-timestamps) lag duration, which determines how far behind the current time CockroachDB will attempt to maintain the closed timestamp. For example, with the default value, if the current time is `12:30:00` then CockroachDB will attempt to keep the closed timestamp at `12:29:57` by aborting or retrying ongoing requests that are below this time.
 
-{{site.data.alerts.callout_danger}}
-Thoroughly test any adjustment in cluster settings before deploying the change in production.
+A changefeed aggregates checkpoints across all ranges, and once the timestamp on all the ranges advances, the changefeed can then [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). In the context of changefeeds, `kv.closed_timestamp.target_duration` affects how old the checkpoints will be, which will determine the latency before changefeeds can consider the history of an event complete.
+
+As a result, adjusting `kv.closed_timestamp.target_duration` can affect changefeeds encountering latency **and** changefeeds causing foreground SQL latency. In clusters running large-scale workloads, it may be helpful to:
+
+- **Decrease** the value for a lower changefeed emission latency — that is, the time from committing a write until the changefeed emits that event.
+- **Increase** the value to reduce the potential impact of changefeeds on SQL latency. This will lower the resource cost of changefeeds, which can be especially important for workloads with tables in the TB range of data.
+
+It is important to note that a changefeed at default configuration does not checkpoint more often than once every 30 seconds. When you create a changefeed with [`CREATE CHANGEFEED`]({% link {{ page.version.version }}/create-changefeed.md %}), you can adjust this with the [`min_checkpoint_frequency`]({% link {{ page.version.version }}/create-changefeed.md %}#min-checkpoint-frequency) option.
+
+#### `kv.rangefeed.closed_timestamp_refresh_interval`
+
+**Default:** `3s`
```

Comment:
The information here is correct for 23.2, but not 23.1.

In 23.1, `kv.rangefeed.closed_timestamp_refresh_interval` defaulted to 0, such that it would fall back to `kv.closed_timestamp.side_transport_interval` (default 200ms). So it is much more aggressive, and costly, by default.

Reply:
The paragraph below still refers to the `3s` value, which is only the default in 23.2.

Consider updating it to mention that the default value of `0` means it falls back to `kv.closed_timestamp.side_transport_interval` at `200ms`, and use that as the basis for the description.

Reply:
Done! Thanks for the detailed reviews Erik — it really helped my understanding of these settings.